eq.yml

```diff
@@ -72,11 +72,15 @@ services:
       REACT_APP_BASE_NAME: /eq-author
       REACT_APP_API_URL: http://localhost:4000/graphql
       REACT_APP_USE_MOCK_API: "false"
+      REACT_APP_PUBLISHER_URL: http://localhost:9000/publish
+      REACT_APP_GO_LAUNCH_A_SURVEY_URL: http://localhost:8000/quick-launch
     depends_on:
       - eq-author-api
+      - eq-survey-launcher
     restart: always
     networks:
       - eq-author
+      - eq-runner
```

Comment:
I don't think you need this as the author doesn't talk directly to runner, it goes via the client.

Reply:
I wasn't sure about this. Is `go-launch-a-suvey` available to `eq-author` on `localhost:8000` even when it's not on the eq-runner network?

Reply:
it doesn't need to be available from author. as author is all running client side, its only the client that needs to access it?

Reply:
Hmm, so does that mean that `REACT_APP_GO_LAUNCH_A_SURVEY_URL` is wrong? localhost:8000 won't resolve to the correct URL from the client?

Reply:
No that is correct. the launcher is bound to port 8000